src/Sarif.Validation/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="Microsoft.CodeAnalysis.SarifValidationLibraries" version="1.0.0" targetFramework="net46" />
   <package id="Microsoft.Json.Schema.ToDotNet" version="0.11.0" targetFramework="net46" />
-  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net46" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
```

Comment:
Shouldn't there be a corresponding change to Sarif.Validation.csproj to update the Newtonsoft.Json version number?